### PR TITLE
feat: Addition of Fields for reCAPTCHA

### PR DESCRIPTION
**Pull request by kushthedude**
References #3129 

#### Short description of what this resolves:

- Addition of fields in the admin console.

image

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@niranjan94 @CosmicCoder96 Please Review

**Comment by kushthedude** [comment]
@niranjan94 @CosmicCoder96 Please Review

File: app/templates/components/forms/admin/settings/system/captcha-option.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<h3 class="ui header">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system/captcha-option.hbs]: rename this component to `captcha-form`

**Comment by kushthedude** [comment]
@CosmicCoder96 did the requested changes, Please review

File: app/templates/components/forms/admin/settings/system-form.hbs [hunk 2]
```diff
@@ -33,10 +33,9 @@
     {{input type='text' name='tag_line' value=settings.tagline}}
   </div>
   <div class="ui hidden divider"></div>
-
   {{forms/admin/settings/system/social-media-token settings=settings}}
-
-
+  <div class="ui hidden divider"></div>
+  {{forms/admin/settings/system/captcha-option settings=settings}}
```
Comment (CosmicCoder96) [inline app/templates/components/forms/admin/settings/system-form.hbs]: you didn't change the name here, it won't work, nothing will get rendered.

Comment (kushthedude) [reply app/templates/components/forms/admin/settings/system-form.hbs]: @CosmicCoder96 Left due to ignorance, Did the change please check

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 did the requested changes, Please review

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 , Please review

- [ ]

**Comment by kushthedude** [comment]
@CosmicCoder96 @niranjan94 , Please review

**Pull request merged**